.github/workflows/snapshot.yml

```diff
@@ -23,21 +23,22 @@ jobs:
       - name: Install dependencies
         run: yarn --frozen-lockfile
 
-      - name: Check if Prerelease is enabled
-        id: is_pre_enabled
-        uses: andstor/file-existence-action@v1
-        with:
-          files: '.changeset/pre.json'
-
-      - name: Enter prerelease mode
-        if: steps.is_pre_enabled.outputs.files_exists == 'false'
-        run: yarn changeset pre enter next
+      - name: Checkout snapshot branch
+        run: |
+          git checkout -b snapshot
+          git reset --hard ${{ github.ref }}
 
       - name: Create release Pull Request or publish to NPM
         id: changesets
         uses: changesets/action@v1
         with:
-          publish: yarn release
+          version: yarn version-packages --snapshot snapshot
+          publish: yarn release --no-git-tags --snapshot
         env:
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Remove snapshot branch
+        run: |
+          git checkout ${{ github.ref }}
+          git branch -D snapshot
```

Comment:
I know I originally proposed this step, but am now wondering if it's needed. A fresh runner is used each time this workflow is triggered and we never push the temp `snapshot-release` branch. Should we delete?

Reply:
🚮